### PR TITLE
Update CSV to add description for the CRDs (release-0.1.0-rc3)

### PR DESCRIPTION
**Pull request by sohankunkerkar**
/cc @abhat

### CODE REVIEW

File: deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -247,7 +247,7 @@ spec:
       kind: KubeFedConfig
       name: kubefedconfigs.core.kubefed.k8s.io
       version: v1beta1
-    - description: IngressDNSRecord
+    - description: A resource that associates one or more Kubernetes Ingress and how to access the Kubernetes Ingress resources, with a scheme for constructing Domain Name System (DNS) resource records for the Ingress.
```
Comment (pmorie) [inline deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml]: These descriptions should be phrased similarly to the other descriptions in this CSV, for example:

"IngressDNSRecord associates one or more Kubernetes Ingress..."

Comment (sohankunkerkar) [reply deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml]: Fixed.

**Comment by sohankunkerkar** [comment]
@pmorie PTAL

**Pull request merged**